KubeArmor/core/kubeUpdate.go

```diff
@@ -486,6 +476,29 @@ func (dm *KubeArmorDaemon) WatchK8sPods() {
 					pod.Labels[k] = v
 				}
 
+				if len(event.Object.ObjectMeta.OwnerReferences) > 0 {
+					// pod is owned by a replicaset, daemonset etc thus we use parent object's name
+					pod.Metadata["ownerName"] = event.Object.ObjectMeta.OwnerReferences[0].Name
```

Comment:
Is it necessary to keep the `ownerName` as part of the `pod.Metadata[]`?
Can we keep it as a local var in the function? I am worried someone might start using `pod.Metadata["ownerName"]` it in the future and it might not be filled up in all cases.